xcodebuild/xcodebuild.go

```diff
@@ -56,23 +58,16 @@ func (b *xcodebuild) Version() (Version, error) {
 
 // Params ...
 type Params struct {
```

Comment:
should not we merge Params and TestParams? originally we differentiated them since the step was running and `xcodebuild build` in case of single_build=true.

Reply:
Good point! (I didn't know they were separated because of this.)

Should we do it in a separate PR?